src/main.c

```diff
@@ -160,7 +161,7 @@ static bool parse_options(int argc, char **argv)
 		case 'x':
 			force = true;
 			break;
-		case 'm':
+		case 'T':
```

Comment:
+1 on the new short option, but you'll want to update the short option list and also the prog_opts array.

Reply:
Ugh, I looked at it like 3 times before agreeing that I updated all the parts. Thanks @phmccarty